qutebrowser/browser/webengine/webview.py

```diff
@@ -310,9 +311,17 @@ def acceptNavigationRequest(self,
             QWebEnginePage.NavigationTypeOther:
                 usertypes.NavigationRequest.Type.other,
         }
+        accept_request = True
+        if(url.scheme() == "qute" and url.path() == "print"):
+            command_dispatcher = objreg.get('command-dispatcher',
+                                            scope='window',
+                                            window=self._win_id)
+            command_dispatcher.printpage()
+            accept_request = False
```

Comment:
This is the wrong place to handle this (`webview.py` should be a pretty minimal wrapper around QWebEngineView). Instead, you should handle it in `webenginetab.py` (in `_on_navigation_request`) where you also already have the `win_id` available.